File: Python/26. Left and Right Sum Differences/sum.py
nums = [1]
leftsum = [len(nums)]
rightsum = []
totalsum = []

leftsum[0] = 0
for i in range(1,len(nums)):
    leftsum.append(leftsum[i-1]+nums[i-1])

sum = 0
for i in range (1,len(nums)):
    sum = nums[i]
    for j in range(i+1,len(nums)):
        sum += nums[j]
    rightsum.append(sum)
    sum = 0
rightsum.append(0)

for i in range(0,len(nums)):
    totalsum.append(abs(leftsum[i] - rightsum[i]))
print(totalsum)

File: Java/5. Sum Multiples/Sum.java
public class Sum {
    public static void main(String[] args) {
        int n = 9;
        int sum = 0;

        for (int i = 0; i < n + 1; i++) {
            if (i % 3 == 0 || i % 5 == 0 || i % 7 == 0) {
                sum = sum + i;
            }
        }
        System.out.println(sum);
    }
}